torchvision/csrc/vision.h

```diff
@@ -14,12 +14,18 @@
 #endif
 #endif
 
+#ifdef VISION_BUILD_LIB
```

Comment:
Same thing is defined here : https://github.com/pytorch/vision/blob/master/torchvision/csrc/cpu/vision_cpu.h#L4
Maybe, we can take the same approach ?

Reply:
Indeed, thanks for noticing :)